tests/plugins/editorplaceholder/editorplaceholder.js

```diff
@@ -205,5 +233,93 @@
 		} );
 	};
 
+	// (#4253)
+	tests[ 'test placeholder loads correctly in full-page editor with bbcode plugin' ] = function() {
```

Comment:
I just noticed this test could be further simplified to something like:

```javascript
	// (#4253)
	tests[ 'test placeholder loads correctly in full-page editor with bbcode plugin' ] = function() {
		addErrorAsserts( this );

		bender.editorBot.create( {
			name: 'bbcodeFailTest',
			config: {
				extraPlugins: 'bbcode',
				editorplaceholder: 'any',
				fullPage: true
			}
		}, function( bot ) {
			// ...
			if ( bot.testCase.consoleErrorStub ) { // Make sure console stub is there (handle older IEs)
				assert.isFalse( bot.testCase.consoleErrorStub.called, 'There should be no errors during initialization' );
			}

			assert.pass( 'Error was not thrown' );
		} );
	};
```

Wait is not needed here, since `resume` in runtime error listener resumes `wait()` from `editorBot.create`.

Reply:
Also you can get rid of additional listener as mentioned in https://github.com/ckeditor/ckeditor4/pull/4370#discussion_r526872559. And handle older IEs console error mentioned in https://github.com/ckeditor/ckeditor4/pull/4370#pullrequestreview-534402832.